yeonjae/programmers-2/스킬트리.js

```diff
@@ -0,0 +1,40 @@
+function solution(skill, skill_trees) {
+  const filteredSkillTrees = [];
+  let answer = 0;
+
+  skill_trees.forEach((el) => {
+    let str = "";
+
+    for (let i = 0; i < el.length; i++) {
+      const idx = skill.indexOf(el[i]);
+
+      if (idx !== -1) {
+        str += el[i];
+      }
+    }
+    filteredSkillTrees.push(str);
+  });
```

Comment:
```suggestion
  const filteredSkillTrees = skill_trees.map((tree) => {
    return tree.split("").filter((el) => skill.includes(el));
  });
```
- 위와 같이 filteredSkillTrees를 map과 filter 등을 이용해서 함수형으로 작성하는게 선언하는게 더 깔끔할 것 같습니다!

Reply:
리팩토링 너무 깔끔한것같습니다! 참고하겠습니다 :)